.gitignore

```diff
@@ -39,3 +39,8 @@ launch-*
 # Local development
 stack.yaml.local.lock
 stack.yaml.local
+
+# latex files
+doc/*.fdb_latexmk
+doc/*.fls
+doc/*.synctex.gz
```

Comment:
When I saw the PR title I had a bit of a panic that you were adding a whole bunch of PDF files to the repo.

Maybe the PR title should have been: "Add files created by doc/Makefile to .gitignore" :)